keps/sig-windows/20190103-windows-node-support.md

```diff
@@ -86,46 +89,84 @@ As of 29-11-2018 much of the work for enabling Windows nodes has already been co
 
 ### What works today
 - Windows-based containers can be created by kubelet, [provided the host OS version matches the container base image](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility)
-    - Pod (single or multiple containers per Pod with process isolation)
-    - Services types NodePort, ClusterIP, LoadBalancer, and ExternalName
+    - Pod (single or multiple containers per Pod with process isolation). There are no notable differences in Pod status fields between Linux and Windows containers
+    - Services types NodePort, ClusterIP, LoadBalancer, and ExternalName. Service environment variables.
     - Workload controllers ReplicaSet, ReplicationController, Deployments, StatefulSets, DaemonSet, Job, CronJob
-    - ConfigMap, Secrets: as environment variables or volumes
+    - ConfigMap, Secrets: as environment variables or volumes (Volume subpath does not work)
```

Comment:
When I was reviewing https://github.com/kubernetes/kubernetes/pull/73204, another difference occurred to me: On Linux, Secret volumes are written to tmpfs, but I assume that they are written unencrypted to persistent storage on Windows, since it doesn't support ram disks?